frontend/public/components/app.jsx

```diff
@@ -218,7 +218,7 @@ const App = (props) => {
   const { productName } = getBrandingDetails();
 
   const content = (
-    <>
+    <HelmetProvider>
```

Comment:
Note that I only added the provider here, so `Helmet` will not work on modals created with the deprecated `createModalLauncher` function (it does work with regular PF modals). 

However, I don't think this is ever done within our codebase, and `createModalLauncher` is not part of the SDK, so I am not sure if this is important or even notable